src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -964,21 +967,21 @@ internal static bool ReportUseSiteDiagnostic(DiagnosticInfo info, DiagnosticBag
         /// <summary>
         /// Derive error info from a type symbol.
         /// </summary>
-        internal bool DeriveUseSiteDiagnosticFromType(ref DiagnosticInfo result, TypeSymbol type)
+        internal bool DeriveUseSiteDiagnosticFromType(ref DiagnosticInfo? result, TypeSymbol? type)
         {
-            DiagnosticInfo info = type.GetUseSiteDiagnostic();
+            DiagnosticInfo? info = type?.GetUseSiteDiagnostic();
```

Comment:
>DiagnosticInfo? info = type?.GetUseSiteDiagnostic(); [](start = 12, length = 52)

I think we are making a change in the wrong place. Could the issue be already fixed by [https://github.com/dotnet/roslyn/pull/38230](https://github.com/dotnet/roslyn/pull/38230)?